service/middleware.go

```diff
@@ -160,6 +160,10 @@ func (uis *UIServer) requireLogin(next http.HandlerFunc) http.HandlerFunc {
 	return requireUser(next, uis.RedirectToLogin)
 }
 
+func (uis *UIServer) requireLoginStatusUnauthorized(next http.HandlerFunc) http.HandlerFunc {
```

Comment:
this doesn't really need to be a method on UI Server

Reply:
it follows the same convention as the other middleware